autovacuum_message_attachment/models/autovacuum_mixin.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2019 Akretion
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+import odoo
+from odoo import api, models
+from odoo.tools.safe_eval import datetime, safe_eval
+
+_logger = logging.getLogger(__name__)
+
+
+class AutovacuumMixin(models.AbstractModel):
+    _name = "autovacuum.mixin"
+    _description = "Mixin used to delete messages or attachments"
+
+    def batch_unlink(self):
+        with api.Environment.manage():
```

Comment:
py.warnings: /usr/lib/python3.10/contextlib.py:135: DeprecationWarning: Since Odoo 15.0, Environment.manage() is useless.

Reply:
removed @CasVissers-360ERP 👍